pkg/manifest/pxetree.go

```diff
@@ -11,27 +11,34 @@ import (
 
 type PXETree struct {
 	Base
-
-	// TODO
-	// - Add compression control (squashfs/erofs, etc.)
+	RootfsCompression string
+	RootfsType        ISORootfsType
 
```

Comment:
Would be super useful to be able to pass level of compression, if that is something osbuild can support. For testing, I would probably set level 1 instead of the default 5 (or 9?) which would speed up build creationg.

But this is completely random, probably material for a different PR.